examples/async/async.bal

```diff
@@ -31,20 +29,42 @@ public function main() {
     io:println(f2.isCancelled());
 
     // async action call
-    future<http:Response|error> f3 = start clientEndpoint->get(
-            "/get?test=123");
+    future<http:Response|error> f3 = start clientEndpoint->get("/get?test=123");
     io:println(sum(25, 75));
     io:println(f3.isDone());
     var response = wait f3;
-    match response {
-        http:Response resp => {
-            io:println(untaint resp.getJsonPayload());
-        }
-        error err => {
-            io:println(err.message);
-        }
+    if (response is http:Response) {
+        io:println(untaint response.getJsonPayload());
+    } else if (response is error) {
+        io:println(response.reason());
     }
     io:println(f3.isDone());
+
+    // Asynchronously call the below functions and wait for either of them to finish
+    future<int> f4 = start square(20);
+    future<string> f5 = start greet("Bert");
+    // You can wait for either of the above asynchronous functions to finish
+    int|string anyResult = wait f4 | f5;
+    // The asynchronous function which finishes first will be taken as the result
```

Comment:
Change "which" to "that" here. Normally "which" is used to add additional information to a sentence and generally you would write it with a comma in front of it. "that" is the correct word to use in this scenario. Apply elsewhere if applicable.